testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/Suite.java

```diff
@@ -17,14 +17,23 @@
 import io.trino.tests.product.launcher.env.configs.ConfigDefault;
 
 import java.util.List;
+import java.util.Optional;
+import java.util.Set;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static io.trino.tests.product.launcher.Configurations.nameForSuiteClass;
 
 public abstract class Suite
 {
+    public static final Optional<Set<String>> ALL_FEATURES = Optional.empty();
+
     public abstract List<SuiteTestRun> getTestRuns(EnvironmentConfig config);
 
+    public Optional<Set<String>> getTestedFeatures(EnvironmentConfig ignoredConfig)
+    {
+        return ALL_FEATURES;
```

Comment:
This should be populated automatically using https://github.com/trinodb/trino/pull/10761, right? Should these two PRs be combined?